.vsconfig

```diff
@@ -6,7 +6,8 @@
     "Microsoft.VisualStudio.Workload.NativeDesktop",
     "Microsoft.VisualStudio.Workload.ManagedDesktop",
     "Microsoft.VisualStudio.Workload.Universal",
-    "Microsoft.VisualStudio.Component.Windows10SDK.18362",
+    "Microsoft.VisualStudio.Component.Windows10SDK.19041",
+    "Microsoft.VisualStudio.Component.Windows10SDK.20348",
```

Comment:
Is this version blocking win10-users from using this feature?

Reply:
no, we only use newer API to disable screen borders during screen measurement. if it turns out unavailable at runtime, we'll just can't disable it, which is not an essential feature

Reply:
Did you try this scenario? Want to be sure that this is not the reason for the crash. (I have Win10 build `10.0.194*`.)

Reply:
I'm on 19041 myself :)